src/Symfony/Bridge/Twig/ErrorRenderer/TwigErrorRenderer.php

```diff
@@ -36,7 +37,7 @@ public function __construct(
         ?HtmlErrorRenderer $fallbackErrorRenderer = null,
         bool|callable $debug = false,
     ) {
-        $this->fallbackErrorRenderer = $fallbackErrorRenderer ?? new HtmlErrorRenderer();
+        $this->fallbackErrorRenderer = \in_array(\PHP_SAPI, ['cli', 'phpdbg', 'embed'], true) ? new CliErrorRenderer() : ($fallbackErrorRenderer ?? new HtmlErrorRenderer());
```

Comment:
Shouldn’t the property type also be updated to support `CliErrorRenderer`? It should be `ErrorRendererInterface`.

Reply:
Yes! Done. Thank you.